templates/configuration/graphdb-jolokia-access-configmap.yaml

```diff
@@ -0,0 +1,16 @@
+# Default configuration map for provisioning GraphDB repository.
+# To change it, prepare another configuration map and update "graphdb.repositoryConfigmap"
+{{- $configs := (.Values.graphdb.configs | default dict) }}
+{{- if $configs.jolokiaAccessConfigMap }}
+{{- if eq $configs.jolokiaAccessConfigMap "graphdb-jolokia-access-configmap" }}
```

Comment:
why not use Values.graphdb.configs.jolokiaAccessConfigMap directly? What's the point to assign it to a variable?

Reply:
if all 5 config maps are commented out .Values.graphdb.configs won't exist, so when you go looking for Values.graphdb.configs.jolokiaAccessConfigMap it will error out. So we make sure it exists, then we make sure $configs.jolokiaAccessConfigMap isn't nil or again it will throw an error when trying to compare it with a string.

Reply:
ok